doc/modules/classes.rst

```diff
@@ -161,28 +163,67 @@ Classes
    :toctree: generated/
    :template: class.rst
 
-   cross_validation.KFold
-   cross_validation.LabelKFold
-   cross_validation.LabelShuffleSplit
-   cross_validation.LeaveOneLabelOut
-   cross_validation.LeaveOneOut
-   cross_validation.LeavePLabelOut
-   cross_validation.LeavePOut
-   cross_validation.PredefinedSplit
-   cross_validation.ShuffleSplit
-   cross_validation.StratifiedKFold
-   cross_validation.StratifiedShuffleSplit
+   model_selection.KFold
+   model_selection.LabelKFold
+   model_selection.StratifiedKFold
+   model_selection.LeaveOneLabelOut
+   model_selection.LeavePLabelOut
+   model_selection.LeaveOneOut
+   model_selection.LeavePOut
+   model_selection.ShuffleSplit
+   model_selection.LabelShuffleSplit
+   model_selection.StratifiedShuffleSplit
+   model_selection.PredefinedSplit
+
+Splitter helper
+---------------
 
+.. currentmodule:: sklearn
 
 .. autosummary::
    :toctree: generated/
    :template: function.rst
 
-   cross_validation.train_test_split
-   cross_validation.cross_val_score
-   cross_validation.cross_val_predict
-   cross_validation.permutation_test_score
-   cross_validation.check_cv
+   model_selection.train_test_split
+
+Parameter tuners
+----------------
+
+.. currentmodule:: sklearn
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   model_selection.GridSearchCV
+   model_selection.RandomizedSearchCV
+   model_selection.ParameterGrid
+   model_selection.ParameterSampler
+
+Scorers
```

Comment:
Scorers are something else.
Maybe collapse these two and call it "Validation" ?